### PR TITLE
CNV docs setup

### PR DESCRIPTION
**Pull request by vikram-redhat**

### CODE REVIEW

File: modules/cnv_introduction-to-cnv.adoc
Comment (vikram-redhat) [inline modules/cnv_introduction-to-cnv.adoc]: @aburdenthehand - use '_' for file names. So this should be cnv_introduction_to_cnv.adoc

File: modules/cnv_introduction_to_cnv.adoc
Comment (vikram-redhat) [inline modules/cnv_introduction_to_cnv.adoc]: @aburdenthehand - use the productname variable as defined in the cnv_document_attributes.adoc file.

**Review by mareklibra — APPROVED** [review]

File: modules/cnv_vm_storage_volume_types.adoc
Comment (ousleyp) [reply modules/cnv_vm_storage_volume_types.adoc]: Fixed in the latest commit.

File: modules/cnv_vm_storage_volume_types.adoc
Comment (ousleyp) [reply modules/cnv_vm_storage_volume_types.adoc]: Fixed in the latest commit.

File: modules/cnv_vm_storage_volume_types.adoc
Comment (aglitke) [inline modules/cnv_vm_storage_volume_types.adoc]: This is really pedantic but when referring to the object or kubernetes type (as you are here) use `PersistentVolumeClaim` and when referring to the VM disk types in the kubevirt API use `persistentVolumeClaim` as that is how it is spelled in the kubevirt API.

**Review by aglitke — COMMENTED** [review]
Looks really good!  My one pedantic comment shows truly how little room for improvement there is! :)

File: modules/cnv_importing_vm_disk_DVs.adoc
Comment (aglitke) [inline modules/cnv_importing_vm_disk_DVs.adoc]: Just noticed that above you say cloning but what we are talking about here is importing...

Comment (ousleyp) [reply modules/cnv_importing_vm_disk_DVs.adoc]: Aha, good catch. No wonder I'm getting all these mixed up. Fixed in:https://github.com/openshift/openshift-docs/pull/12518/commits/4e71a10ddff35d1d0d230ce8471c81b8992919c4

File: modules/cnv_vm_storage_volume_types.adoc
Comment (ousleyp) [reply modules/cnv_vm_storage_volume_types.adoc]: Pedantic comments are always welcome. I *think* I fixed this here (same commit as the titling issue): https://github.com/openshift/openshift-docs/pull/12518/commits/4e71a10ddff35d1d0d230ce8471c81b8992919c4

**Review by ousleyp — COMMENTED** [review]
I like this new world with two different Reference sections. It's a lot cleaner.

File: modules/cnv_template_import_dv.adoc
Comment (aglitke) [inline modules/cnv_template_import_dv.adoc]: This would not be large enough to accommodate the download.  Change to 1G.

File: modules/cnv_template_clone_DV.adoc
Comment (aglitke) [inline modules/cnv_template_clone_DV.adoc]: I'd suggest to use 1G here as it's closer to the scale of sizes we'd actually be using.

Comment (ousleyp) [reply modules/cnv_template_clone_DV.adoc]: Fixed: https://github.com/openshift/openshift-docs/pull/12518/commits/8f994e43c293c237fe3c2192c2f6aa220c163b8c

File: modules/cnv_template_import_dv.adoc
Comment (ousleyp) [reply modules/cnv_template_import_dv.adoc]: Fixed: https://github.com/openshift/openshift-docs/pull/12518/commits/8f994e43c293c237fe3c2192c2f6aa220c163b8c

**Review by fabiand — APPROVED** [review]
lgtm

**Comment by vikram-redhat** [comment]
Thanks @aburdenthehand and @ousleyp .

@openshift/team-documentation PTAL - I have removed the WIP from the PR, and this PR has been QE reviewed already. Andrew and Pan realize they will need to squash the commits before merging but are waiting on the peer review to incorporate those changes in.

File: cnv_release_notes/cnv_release_notes.adoc
Comment (nellyc) [inline cnv_release_notes/cnv_release_notes.adoc]: 'may still' is not accurate, 
it will always boot from PXE, until you stop and start the vm

File: modules/cnv_controlling_vms_webconsole.adoc
Comment (kalexand-rh) [inline modules/cnv_controlling_vms_webconsole.adoc]: This procedure isn't used. Please remove it.

File: modules/cnv_creating_cluster_admin_user.adoc
Comment (kalexand-rh) [inline modules/cnv_creating_cluster_admin_user.adoc]: Should "Container-native Virtualization" be a reused variable?

**Review by kalexand-rh — COMMENTED** [review]
One batch of comments, questions, picks and suggestions. (I haven't reviewed all the content yet.)

I think you need to change all your module anchors.

**Review by kalexand-rh — COMMENTED** [review]
More comments

**Comment by kalexand-rh** [comment]
I got tired of marking things up, so here are some general requests:
- Replace Container-native Virtualization with the {ProductName}.
- Replace OpenShift and all of its variants with {product-title}
- Remove all internal links and xrefs.
- Update all UI items to be offset with *s.
- Remove all instances of "button" and adjust articles as required.
- Remove all instances of should, would, could, and may.
- Check that all instance of "will" are required. Will the new state be introduced quickly enough that "is" is sufficient, or do you need the future tense for clarity?
- Remove the numbers in OLs.
- It's not a "drop-down menu." It's a list.
- Your heading cases are inconsistent. Please pick one.
- Don't use "we" or "our."
- Don't use "please."
- It's an "inventory file" not an "inventory."
- Use simple, present tense wherever possible.

File: modules/cnv_installing_web_console.adoc
Comment (ousleyp) [reply modules/cnv_installing_web_console.adoc]: Is this what you meant, or were you referring to the link? 

"SSH into the master node as the `ansible_ssh_user` defined in the inventory file for {ProductName}."

File: modules/cnv_install_cnv_apb.adoc
Comment (ousleyp) [reply modules/cnv_install_cnv_apb.adoc]: I think "plaintext" is the correct term for unencrypted text (as opposed to unformatted text). Changed to "which allows" - thanks!

File: modules/cnv_install_cnv_apb.adoc
Comment (ousleyp) [reply modules/cnv_install_cnv_apb.adoc]: Yeah, seems pretty important. :) Removed "(optional)"

File: modules/cnv_controlling_vms.adoc
Comment (ousleyp) [reply modules/cnv_controlling_vms.adoc]: I'm really not sure, because this was from @aburdenthehand's original docs (pre-modularization). We'll have to revisit this later. :)

**Comment by vikram-redhat** [comment]
This branch has been rebased on the master. All commits (125!!) have been squashed.

Good work @ousleyp and @aburdenthehand .

@kalexand-rh - this is ready to be merged and cherrypicked when you get the green light from @knrc 

cc: @aheslin

(I can't seem to tag Audrey or Michelle).

**Comment by knrc** [comment]
@vikram-redhat I don't think you meant to include me in this issue

File: modules/cnv_creating_cluster_admin_user.adoc
Comment (jcpowermac) [inline modules/cnv_creating_cluster_admin_user.adoc]: I ran through this documentation yesterday as part of an exercise of installing the released CNV bits.  This section is confusing.  If the `cnv-admin` does not exist in the configured `identityProvider` first you cannot login because there is no configured password.  The `Additional Resources` section referencing the identity providers should be before login.

Comment (aburdenthehand) [reply modules/cnv_creating_cluster_admin_user.adoc]: Thanks for the comment. I've updated the content (moved existing link to a prereq and changed the step). I'd be much obliged if you could let me know if it's an improvement or not.

Comment (jcpowermac) [reply modules/cnv_creating_cluster_admin_user.adoc]: @aburdenthehand so much better!

File: modules/cnv_template_openshift_inventory.adoc
Comment (simon3z) [inline modules/cnv_template_openshift_inventory.adoc]: @ousleyp it seems there's a special character here to remove.

Comment (ousleyp) [reply modules/cnv_template_openshift_inventory.adoc]: I retyped the line and Nelly confirmed it appears to be gone now. Thanks Federico!

**Pull request merged**
